fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-unused-vars -- Remove this line. */
+import React, { useState, useEffect } from 'react';
+import UserList from './user-list';
+
+export default function App() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [users, setUsers] = useState([]);
+  const [error, setError] = useState();
+
+  /* your code here (hint: useEffect) */
+  useEffect(() => {
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then((response) => response.json())
+      .then((data) => {
+        setUsers(data);
+      })
+      .catch((error) => {
+        console.error('There was an error!', error);
+        setError(error);
+      });
+    setIsLoading(false);
```

Comment:
`fetch` returns a `Promise`. Please make sure you don't turn off loading while that `Promise` is still pending.